app/pb_kits/playbook/pb_weekday_stacked/_weekday_stacked.scss

```diff
@@ -0,0 +1,17 @@
+@import "../pb_caption/caption"; //used for weekday (caption)
+@import "../pb_title/title";   //used for date (title 1)
+
+[class^=pb_weekday_stacked_kit] {
+    &[class*=_center]  {
+      & > [class^=pb_caption],
+      & > [class^=pb_title] {
+        text-align: center;
```

Comment:
Although `& > [class^=pb_caption]` works to target the nested kits, it is potentially dangerous to target the dynamic class.  A better way to target the nested kits is like below:

Add a custom class through the prop `classname`, and use the custom class within your scss.

```ruby
def weekday
  if is_set? timestamp_weekday
    pb_weekday = Playbook::PbCaption::Caption.new(text: timestamp_weekday, dark: dark, classname: my_custom_class)
    ApplicationController.renderer.render(partial: pb_weekday, as: :object)
  end
end
```

```scss 
[class^=pb_weekday_stacked_kit] {
    &[class*=_center]  {
        .my_custom_class {
              ...
         }
    }
}
```